react/components/ProductPrice/index.js

```diff
@@ -225,6 +228,12 @@ priceWithIntel.schema = {
       default: Price.defaultProps.labelSellingPrice,
       isLayout: true,
     },
+    labelListPrice: {
+      type: 'string',
+      title: 'admin/editor.productPrice.labelListPrice',
+      default: Price.defaultProps.labelListPrice,
+      isLayout: true,
```

Comment:
Since that might be a translated text, this should not be a layout prop. The same applies for the other labels here.

Reply:
Should I change all the props to `isLayout: false,`?